formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java

```diff
@@ -44,7 +44,7 @@ public final class JsonFormat implements EventFormat {
     /**
      * Content type associated with the JSON event format
      */
-    public static final String CONTENT_TYPE = ContentType.JSON.value();
+    public static final String CONTENT_TYPE = "application/cloudevents+json";
```

Comment:
> "_&hellip;JsonFormat.CONTENT_TYPE == ContentType.JSON.value(), can we replace the hardcoded string with that?&hellip;_"

I've rolled back `JsonFormat.java` to the hardcoded string, @pierDipi. That repairs [_that build failure_](https://github.com/cloudevents/sdk-java/actions/runs/4599824636/jobs/8137958743#step:4:7108).

> "_&hellip;Same for ProtobufFormat.PROTO_CONTENT_TYPE and XMLFormat.XML_CONTENT_TYPE&hellip;_"

I've left those other two as `public static final String X_TYPE = ContentType.X.value()` though.

Afterwards, I ran `mvn -verify` in my branch. Locally, everything compiles and all tests pass.

Reply:
Ok, can you do the same for xml and proto?